src/ui/templates/insurance/your-business/company-details.njk

```diff
@@ -98,109 +98,154 @@
+            'data-cy': 'company-website-heading'
+          }
           },
+          id: FIELDS.COMPANY_DETAILS.WEBSITE,
+          name: FIELDS.COMPANY_DETAILS.WEBSITE,
+          classes: "input-width",
           attributes: {
-            'data-cy': 'trading-address'
+            'data-cy': 'company-website',
+            'aria-label': CONTENT_STRINGS.WEBSITE
           }
-        },
-        items: [
-          {
-            value: "yes",
-            text: "Yes",
+        }) }}
+      </div>
+
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+        {{ govukInput({
+          label: {
+            text: CONTENT_STRINGS.PHONE_NUMBER,
+            classes: "govuk-label--m",
+            isPageHeading: false,
             attributes: {
-              'data-cy': "trading-address-yes"
+            'data-cy': 'company-phone-heading'
             }
           },
-          {
-            value: "no",
-            text: "No",
+          hint: {
+            text: CONTENT_STRINGS.PHONE_NUMBER_HINT,
             attributes: {
-              'data-cy': "trading-address-no"
+              'data-cy': 'company-phone-hint'
             }
-          }
-        ]
-      }) }}
-    </div>
-
-    <div class="govuk-grid-column-three-quarters-from-desktop">
-      {{ govukInput({
-        label: {
-          text: CONTENT_STRINGS.WEBSITE,
-          classes: "govuk-label--m",
-          isPageHeading: false,
-          attributes: {
-          'data-cy': 'company-website-heading'
-        }
-        },
-        id: "company-website",
-        name: "company-website",
-        classes: "input-width",
-        attributes: {
-          'data-cy': 'company-website'
-        }
-      }) }}
-    </div>
-
-    <div class="govuk-grid-column-three-quarters-from-desktop">
-      <div class="govuk-button-group">
-        {{ govukButton({
-          text: CONTENT_STRINGS.BUTTONS.CONTINUE,
+          },
+          id: FIELDS.COMPANY_DETAILS.PHONE_NUMBER,
+          name: FIELDS.COMPANY_DETAILS.PHONE_NUMBER,
+          classes: "input-width",
           attributes: {
-            'data-cy': 'continue-button'
+            'data-cy': 'company-phone',
+            'aria-label': CONTENT_STRINGS.PHONE_NUMBER
           }
         }) }}
+      </div>
 
-        {{ govukButton({
-            text: CONTENT_STRINGS.BUTTONS.SAVE_AND_BACK,
-            classes: "govuk-button--secondary",
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: CONTENT_STRINGS.BUTTONS.CONTINUE,
             attributes: {
-              'data-cy': 'save-and-back-button'
+              'data-cy': 'continue-button'
             }
-        }) }}
+          }) }}
+
+          {{ govukButton({
+              text: CONTENT_STRINGS.BUTTONS.SAVE_AND_BACK,
+              classes: "govuk-button--secondary",
+              attributes: {
+                'data-cy': 'save-and-back-button'
```

Comment:
Please add `preventDoubleClick: true`

Reply:
This is already dealt with globally 😎  https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/scripts/govuk-frontend.js#L5